controllers/apply-next/views/view-helpers.njk

```diff
@@ -65,3 +67,20 @@
         </div>
     {% endfor %}
 {% endmacro %}
+
+
+{% macro showSectionProgress(sectionProgress) %}
+    {% if sectionProgress %}
+        <div class="application-summary__status__label u-padded-vertical-s">
```

Comment:
Makes sense for this to be an external macro, but I'd try and pull the `application-summary__status__label` out of the macro to avoid having a BEM fragment in a macro.